mender-update/cli/actions.cpp

```diff
@@ -14,12 +14,23 @@
 
 #include <mender-update/cli/actions.hpp>
 
+#include <algorithm>
+#include <iostream>
+#include <string>
+
+#include <common/common.hpp>
+#include <common/error.hpp>
 #include <common/events.hpp>
+#include <common/key_value_database.hpp>
 #include <common/log.hpp>
+#include <common/path.hpp>
 
+#include <mender-update/cli/cli.hpp>
 #include <mender-update/daemon.hpp>
 #include <mender-update/standalone.hpp>
 
+#include <artifact/config.hpp>
```

Comment:
I guess it could go along with the `mender-update` ones, but I thought I'd separate them. but no biggie, I'll do whatever yous think makes the most sense.